dbms/src/main/java/ch/unibas/dmi/dbis/polyphenydb/processing/VolcanoQueryProcessor.java

```diff
@@ -55,6 +77,63 @@
+                    EnumerableRules.ENUMERABLE_JOIN_RULE,
+                    EnumerableRules.ENUMERABLE_MERGE_JOIN_RULE,
+                    EnumerableRules.ENUMERABLE_SEMI_JOIN_RULE,
+                    EnumerableRules.ENUMERABLE_CORRELATE_RULE,
+                    EnumerableRules.ENUMERABLE_PROJECT_RULE,
+                    EnumerableRules.ENUMERABLE_FILTER_RULE,
+                    EnumerableRules.ENUMERABLE_AGGREGATE_RULE,
+                    EnumerableRules.ENUMERABLE_SORT_RULE,
+                    EnumerableRules.ENUMERABLE_LIMIT_RULE,
+                    EnumerableRules.ENUMERABLE_COLLECT_RULE,
+                    EnumerableRules.ENUMERABLE_UNCOLLECT_RULE,
+                    EnumerableRules.ENUMERABLE_UNION_RULE,
+                    EnumerableRules.ENUMERABLE_INTERSECT_RULE,
+                    EnumerableRules.ENUMERABLE_MINUS_RULE,
+                    EnumerableRules.ENUMERABLE_TABLE_MODIFICATION_RULE,
+                    EnumerableRules.ENUMERABLE_VALUES_RULE,
+                    EnumerableRules.ENUMERABLE_WINDOW_RULE,
+                    EnumerableRules.ENUMERABLE_TABLE_SCAN_RULE,
+                    EnumerableRules.ENUMERABLE_TABLE_FUNCTION_SCAN_RULE );
+
+    public static final List<RelOptRule> DEFAULT_RULES =
+            ImmutableList.of(
+                    TableScanRule.INSTANCE,
+                    RuntimeConfig.JOIN_COMMUTE.getBoolean()
+                            ? JoinAssociateRule.INSTANCE
+                            : ProjectMergeRule.INSTANCE,
+                    FilterTableScanRule.INSTANCE,
+                    ProjectFilterTransposeRule.INSTANCE,
+                    FilterProjectTransposeRule.INSTANCE,
+                    FilterJoinRule.FILTER_ON_JOIN,
+                    JoinPushExpressionsRule.INSTANCE,
+                    AggregateExpandDistinctAggregatesRule.INSTANCE,
+                    AggregateReduceFunctionsRule.INSTANCE,
+                    FilterAggregateTransposeRule.INSTANCE,
+                    ProjectWindowTransposeRule.INSTANCE,
+                    JoinCommuteRule.INSTANCE,
+                    JoinPushThroughJoinRule.RIGHT,
+                    JoinPushThroughJoinRule.LEFT,
+                    SortProjectTransposeRule.INSTANCE,
+                    SortJoinTransposeRule.INSTANCE,
+                    SortRemoveConstantKeysRule.INSTANCE,
+                    SortUnionTransposeRule.INSTANCE );
+
+    public static final List<RelOptRule> CONSTANT_REDUCTION_RULES =
+            ImmutableList.of(
+                    ReduceExpressionsRule.PROJECT_INSTANCE,
+                    ReduceExpressionsRule.FILTER_INSTANCE,
+                    ReduceExpressionsRule.CALC_INSTANCE,
+                    ReduceExpressionsRule.JOIN_INSTANCE,
+                    ValuesReduceRule.FILTER_INSTANCE,
+                    ValuesReduceRule.PROJECT_FILTER_INSTANCE,
+                    ValuesReduceRule.PROJECT_INSTANCE,
+                    AggregateValuesRule.INSTANCE );
+
+
```

Comment:
Could you explain why this changes was made?

Reply:
Simply refactoring. These rules have before been defined in another class.